pkgs/tools/misc/coreutils/default.nix

```diff
@@ -59,6 +59,10 @@ stdenv.mkDerivation rec {
     # Heap buffer overflow that's been here since coreutils 7.2 in 2009:
     # https://www.openwall.com/lists/oss-security/2025/05/27/2
     ./CVE-2025-5278.patch
+
+    # Fixes test-float-h failure on ppc64 with C23
+    # https://lists.gnu.org/archive/html/bug-gnulib/2025-07/msg00021.html
+    ./gnulib-float-h-tests-port-to-C23-PowerPC-GCC.patch
```

Comment:
Can we now fetchpatch the upstream fixes from Savannah?

Reply:
I don't think so?

https://github.com/NixOS/nixpkgs/blob/1bd5a8fb7abc74d461383f4841a6b3aaa047d2ed/pkgs/tools/misc/coreutils/default.nix#L31-L33

Reply:
I had to modify each of them because

- Sometimes `gnulib` files are all vendored into `gnulib-tests`, sometimes across `lib` and `gnulib-tests`
- The test needs to link against libm now. The way gnulib handles this in their repo is via some autoconf/automake module magic which eventually looks for & adds `-lm` in the generated Makefile. The versions vendored & pre-comfigured into other projects don't seem to use this system AFAICT, so I manually added `-lm` to the Makefile templates. And the hunk from one patch file wouldn't apply when used in another project cus all of them are slightly different, hence one slightly different patch per project.

Want me to add a note about this in the comments?

Reply:
Yes please!

Reply:
Added information about which commits are squashed together, and which adjustments were necessary, to the header of each patch file, and a comment in the Nix code with a brief summary + "see header".